profiler-optional/profiler-optional-jdk8/src/main/java-oracle/com/sun/management/OperatingSystemMXBean.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 Naver Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.sun.management;
+
+/**
+ * @author Roy Kim
+ */
+public interface OperatingSystemMXBean extends java.lang.management.OperatingSystemMXBean {
```

Comment:
Shouldn't this be in profiler-optional-jdk6 module?

Reply:
Placed to profiler-optional-jdk6 module?